README.md

```diff
@@ -35,6 +35,20 @@ Collect translatable strings and create `*.pot` files.
 ## check:pot
 Check for common mistakes in translated texts.
 
+## check:spelling
+Checks spelling in `*.md` and `*.html` files. It uses a global custom dictionary
+(file `lib/tasks/spell.dict` in this repository) and supports repository specific
+dictionary (`spell.dict` file in the repository root directory).
```

Comment:
I think @mvidner comment is about having `lib/tasks/spell.dict` ( not `lib/tasks/.spell.dict`) and `.spell.dict` in top level repository for specific stuff.